frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -454,6 +467,22 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
         ],
     }),
     listeners(({ props, values, actions, cache }) => ({
+        playerErrorSeen: ({ errorEvent }) => {
+            const fingerprint = encodeURIComponent(
+                errorEvent.message + errorEvent.filename + errorEvent.lineno + errorEvent.colno
+            )
```

Comment:
Isn't all this info minified at runtime? or is that the session's content?

Reply:
I _think_ this stops me spamming multiple error messages repeating per recording.

Running locally I can see sensible information appearing here... but 🤞